File: CODES/2) PYTHON/2) INTEGER/23.py
n = int(input("n = "))
soat = n // 3600
n = n % 3600
min = n // 60
n = n % 60
sek = n % 3600
print(soat, " soat ")
print(min, " minut ")
print(sek, " sekund ")

File: LANGUAGES/4) Php/2) INTEGER/23.php
<?php
$n = 3700;
echo  $n . " sekund = ";
$soat = intval($n / 3600);
$n = $n % 3600;
$min = intval($n / 60);
$sekund = $n % 60;
echo $soat . " soat " . $min . " minut " . $sekund . " sekund ";
